app/client/templates/elements/accountLink.js

```diff
@@ -13,6 +13,6 @@ Template['elements_account_link'].helpers({
     @method (getAccount)
     */
     'getAccount': function(){
-        return Helpers.getAccountByAddress(this.address) || {address: this.address};
+        return Helpers.getAccountByAddress(this.address) || {address: Helpers.checksummedAddress(this.address)};
```

Comment:
Not sure where the getAccount is used, but you can now use

``{{toChecksumAddress getAccount}}``

This will also fix the issue, you have where you don't show the address formatted when it finds the `Helpers.getAccountByAddress(this.address)` (which would currently not show the formatted address)